components/invoice.js

```diff
@@ -232,8 +232,9 @@ export const useInvoiceable = (onSubmit, options = defaultOptions) => {
   // this function will be called before the Form's onSubmit handler is called
   // and the form must include `cost` or `amount` as a value
   const onSubmitWrapper = useCallback(async (formValues, ...submitArgs) => {
-    let { cost, amount } = formValues
+    let { cost, imageFeesInfo, amount } = formValues
     cost ??= amount
+    if (imageFeesInfo?.totalFees) cost += imageFeesInfo.totalFees
```

Comment:
Should this function used throughout the code be aware of images at all? Could the calling context do this calculation instead?

Reply:
> Could the calling context do this calculation instead?

Probably. I was thinking about using React Context more. So for example, we could wrap `useFormik` and expose the text input value via a wrapper hook like `useText` and the cost field via `useCost`. I was also thinking about creating a `useCursor` for cursor based operations. This might make things easier? :thinking: 

But need to check. This might be more "unrelated" changes than I initially wanted to include in this PR though.

But I can do it and then see which changes I'll put in this PR and which into a different, stacked PR :)

Reply:
Decided to simply calculate the total fees in the calling context now in 4b4da33